src/shell/shell_batch.cpp

```diff
@@ -129,12 +138,7 @@ bool BatchFile::ReadLine(char * line) {
 				next -= '0';
 				if (cmd->GetCount()<(unsigned int)next) continue;
 				std::string word;
-				if (!cmd->FindCommand(next,word)) continue;
```

Comment:
Accidental removal I mentioned in one on comments for next commit.

Reply:
Re-added it to this commit; thanks @dreamer .